File: app/src/main/java/uk/co/ourfriendirony/medianotifier/activities/ActivityUnplayed.java
package uk.co.ourfriendirony.medianotifier.activities;

import android.app.NotificationManager;
import android.content.Context;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.viewpager.widget.PagerAdapter;
import androidx.viewpager.widget.ViewPager;

import uk.co.ourfriendirony.medianotifier.R;
import uk.co.ourfriendirony.medianotifier.activities.pageradapter.UnplayedPagerAdapter;

import static uk.co.ourfriendirony.medianotifier.general.Constants.INTENT_KEY;

public class ActivityUnplayed extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_pager);

        String intentKey = getIntent().getExtras().getString(INTENT_KEY);
        getSupportActionBar().setTitle("Released " + intentKey + "s");
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        cancelNotifications();

        ViewPager mPager = findViewById(R.id.pager);
        PagerAdapter mPagerAdapter = new UnplayedPagerAdapter(getSupportFragmentManager(), intentKey);
        mPager.setAdapter(mPagerAdapter);
        mPager.setCurrentItem(0);
    }

    private void cancelNotifications() {
        NotificationManager notificationManager = (NotificationManager) getApplicationContext().getSystemService(Context.NOTIFICATION_SERVICE);
        notificationManager.cancelAll();
    }
}

File: app/src/main/java/uk/co/ourfriendirony/medianotifier/activities/ActivityUnplayed.kt
package uk.co.ourfriendirony.medianotifier.activities

import android.app.NotificationManager
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.viewpager.widget.PagerAdapter
import androidx.viewpager.widget.ViewPager
import uk.co.ourfriendirony.medianotifier.R
import uk.co.ourfriendirony.medianotifier.activities.pageradapter.UnplayedPagerAdapter
import uk.co.ourfriendirony.medianotifier.general.Constants.INTENT_KEY

class ActivityUnplayed : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_pager)
        val intentKey = intent.extras!!.getString(INTENT_KEY)
        supportActionBar!!.title = "Released " + intentKey + "s"
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        cancelNotifications()
        val mPager = findViewById<ViewPager>(R.id.pager)
        val mPagerAdapter: PagerAdapter = UnplayedPagerAdapter(supportFragmentManager, intentKey)
        mPager.adapter = mPagerAdapter
        mPager.currentItem = 0
    }

    private fun cancelNotifications() {
        val notificationManager = applicationContext.getSystemService(NOTIFICATION_SERVICE) as NotificationManager
        notificationManager.cancelAll()
    }
}